docs/data-cookbooks/cnm-workflow.md

```diff
@@ -0,0 +1,470 @@
+
+Prior to working through this entry you should be familiar with the [Cloud Notification Mechanism](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cloud+Notification+Mechanism).
+
+## Prerequisites
+
+#### Cumulus
+
+This entry assumes you have a deployed instance of Cumulus (>= version 1.8).
+
+#### AWS CLI
+
+This entry assumes you have the [AWS CLI](https://aws.amazon.com/cli/) installed and configured.   If you do not, please take a moment to review the documentation (particularly the [examples relevant to Kinesis](https://docs.aws.amazon.com/streams/latest/dev/fundamental-stream.html) and install it now.
+
+#### Kinesis
+
+This entry assumes you already have two [Kinesis](https://aws.amazon.com/kinesis/) data steams created for use as CNM notification and response data streams.
+
+If you do not have two streams setup, please take a moment to review the [Kinesis documentation](https://aws.amazon.com/documentation/kinesis/) and setup two basic single-shard streams for this example:
+
+Using the "Create Data Stream" button on the [Kinesis Dashboard](https://console.aws.amazon.com/kinesis/home), work through the dialogue.
+
+You should be able to quickly use the "Create Data Stream" button on the [Kinesis Dashboard](https://console.aws.amazon.com/kinesis/home), and setup streams that are similar to the following example:
+
+![](../images/cnm_create_kinesis_stream.jpg)
+
+Please bear in mind that your `${deployment}-lambda-processing` IAM role will need permissions to write to the response stream for this workflow to succeed if you create the Kinesis stream with a dashboard user.  The most straightforward approach is to attach the `AmazonKinesisFullAccess` policy for the stream resource, however your environment/security policies may require an approach specific to your deployment environment.  In operational environments it's likely science data providers would typically be responsible for providing a Kinesis stream with the appropraite permissions.
+
+For more information on how this process works and how to develop a process that will add records to a stream, read the [Kinesis documentation](https://aws.amazon.com/documentation/kinesis/) and the [developer guide](https://docs.aws.amazon.com/streams/latest/dev/introduction.html).
+
+#### Ingest Source Data
+
+This entry will run the SyncGranule task against a single target data file.  To that end it will require a single data file to be present in an S3 bucket matching the Provider configured in the next section.
+
+#### Collection and Provider
+
+Cumulus will need to be configured with a Collection and Provider entry of your choosing.  The provider should match the location of the source data from the `Ingest Source Data` section.
+
+This can be done via the [Cumulus Dashboard](https://github.com/nasa/cumulus-dashboard) if installed or the [API](../api.md).  It is strongly recommended to use the dashboard if possible.
```

Comment:
"Who recommends the dashboard over api? and why?" 

I do, and hope the team agrees, because it's probably easier for exploratory users to interface with the API via the dashboard than fiddle with auth tokens.